docs/site/angular-auth-oidc-client/docs/migrations/v11-to-v12.md

```diff
@@ -281,137 +146,318 @@ export const httpLoaderFactory = (httpClient: HttpClient) => {
-      .subscribe((result: OidcClientNotification<AuthorizationResult>) => {
-        console.log('NewAuthorizationResult', result);
-        console.log('isAuthenticated', result.value.authorizationState === AuthorizedState.Authorized);
-      });
-  }
+```ts
+export interface OpenIdConfiguration {
+  autoUserinfo?: boolean;
+  // ...
 }
 ```
 
-### New
+New:
 
-```typescript
-import { Component, OnInit } from '@angular/core';
-import { PublicEventsService, EventTypes, OidcClientNotification, AuthorizationResult } from 'angular-auth-oidc-client';
-import { filter } from 'rxjs/operators';
-
-@Component({
-  selector: 'app-root',
-  templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss'],
-})
-export class AppComponent implements OnInit {
-  title = 'AuthTest';
-  constructor(private readonly eventService: PublicEventsService) {}
-
-  ngOnInit(): void {
-    this.eventService
-      .registerForEvents()
-      .pipe(filter((notification) => notification.type === EventTypes.NewAuthorizationResult))
-      .subscribe((result: OidcClientNotification<AuthorizationResult>) => {
-        console.log('NewAuthorizationResult', result);
-        console.log('isAuthenticated', result.value.isAuthenticated);
-      });
-  }
+```ts
+export interface OpenIdConfiguration {
+  autoUserInfo?: boolean;
+  // ...
 }
 ```
 
-### `logoff` expects object now
+### Custom Params to pass to requests have been renamed
 
-The parameter to be passed into the `logoff` method is now an object with the properties `urlHandler` and `customParams` which provides you to pass custom parameters to the logoff url.
+Old:
 
-#### Old
+```ts
+export interface OpenIdConfiguration {
+  customParams?: { [key: string]: string | number | boolean };
+  customParamsRefreshToken?: { [key: string]: string | number | boolean };
+  customParamsEndSession?: { [key: string]: string | number | boolean };
+  customTokenParams?: { [key: string]: string | number | boolean };
+  // ...
+}
+```
 
-```typescript
-logoff(urlHandler?: (url: string) => any) {
-  return this.logoffRevocationService.logoff(urlHandler);
+New:
+
+```ts
+export interface OpenIdConfiguration {
+  customParamsAuthRequest?: { [key: string]: string | number | boolean };
+  customParamsRefreshTokenRequest?: { [key: string]: string | number | boolean };
+  customParamsEndSessionRequest?: { [key: string]: string | number | boolean };
+  customParamsCodeRequest?: { [key: string]: string | number | boolean };
+  // ...
+}
+```
+
+## Service `OidcSecurityService`
+
+### Getter for `configuration` and object `PublicConfiguration` replaced by method
+
+The getter for the active configuration was removed as well as the object `PublicConfiguration`. As the `AuthWellKnownEndpoints` are now part of the config the property was replaced with methods to get the configuration directly. Please read below how to get the current used configuration.

```

Comment:
Minor grammatical mistakes make this a bit confusing. I would correct like so:

The getter for the active configuration was removed as well as the object `PublicConfiguration`. Since the `AuthWellKnownEndpoints` parameter is now part of the config, the property was replaced with methods to get the configuration directly. Please read below **on** how to get the current**ly** used configuration.